### DISCUSSION TITLE
diary/week5/

### DISCUSSION THREAD

**Discussion by giscus** [SunnyCloudYang/SunnyCloudYang.github.io#19] [Comments]
# diary/week5/

23.02.27 ~ 23.03.02

https://sunnycloudyang.github.io/diary/week5/

**Comment by SunnyCloudYang** [comment]
>上次发现有个机器上有个叫“EMO”的按钮

EMO，指Emergency Off，急停按钮。[EMO和EMS的区别](https://machinerysafety101.com/2022/02/08/emo-vs-estop/)。
